SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -61,10 +61,31 @@ class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker")
 	bool bEnableMultiWorker;
 
-public:
 	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
 	TSubclassOf<USpatialMultiWorkerSettings> MultiWorkerSettingsClass;
 
+	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
+	TSubclassOf<USpatialMultiWorkerSettings> EditorMultiWorkerSettingsClass;
```

Comment:
Could we slightly name these to make it clearer it's an override, I'm thinking something like?

```suggestion
	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
	TSubclassOf<USpatialMultiWorkerSettings> MultiWorkerSettings;

	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
	TSubclassOf<USpatialMultiWorkerSettings> EditorMultiWorkerSettingsOverride;
```

Reply:
Good point. Updated name as suggested.